example-get-started-experiments/code/README.md

```diff
@@ -50,9 +50,12 @@ You can run [`dvc pull`](https://man.dvc.org/pull) to download the data:
 ```console
 $ dvc pull
 ```
+If you get an error when you `dvc pull`, don't worry you will still be able to run your experiments, this is because (of DVC shared cache? guessinghttps://dvc.org/doc/user-guide/how-to/share-a-dvc-cache#configure-the-shared-cache)
```

Comment:
I don't think we need this. There should be no errors. It's a bug if that is not the case.

Reply:
@alex000kim Can explain more.  We intend to use this project for the DVC Live Metircs release and when you follow the instructions of the README, this error occurs.  So do we need to hold this until this is not the case?

Reply:
There used to be an issue with `dvc pull`, but it fixed now: https://github.com/iterative/example-repos-dev/issues/179

Reply:
Yay!  🙏